src/50_Samples_%26_Templates/Product.html

```diff
@@ -173,27 +120,22 @@
     <script async src="https://cdn.ampproject.org/v0.js"></script>
     </head>
     <body>
-        <!--
-        Use `amp-img` to add the logo of the seller.
-        -->
-        <div class="seller-logo">
-            <amp-img src="/img/google_store.png" width="163" height="35"></amp-img>
-        </div>
+    <%>product/header.html%>
 
         <!-- This button adds the item to the shopping cart. Pressing this button should lead to a the shopping cart page on your normal website and the cart should be prefilled with the current item. -->
         <div id="product-header">
-            <h2>Nexus 6P</h2>
-            <a href="/shopping-cart-sample.html" class="button button-primary call-to-action seller-button">Buy</a>
+            <h2>Apple</h2>
+            <a href="/shopping-cart-sample.html" class="button button-primary call-to-action">Buy</a>
```

Comment:
shopping-cart-sample.html is not an AMP page. Should we do something similar to what we did with [amp-form-success](https://github.com/ampproject/amp-by-example/blob/master/src/amp-form-success.html)?